### ISSUE TITLE
Automating integration tests

### ISSUE THREAD

**Issue by melvyniandrag** [nltk/nltk#1849]
I wanted to make a few contributions to the library a year or two ago but I couldn't get Jenkins working and just gave up. I just made a PR to tensorflow, and jenkins ran automatically! I don't know how they did that, but that seems like a good thing to have!

I don't know if that automation is something I could contribute, or if anyone has given this any thought? I don't even know what would be required to do this? There are definitely all kinds of hairy details about configuration, and probably a server would have to be rented, which would cost money. Maybe its a nonstarter but it seems nice.

**Comment by stevenbird** [comment]
Your message isn't quite clear about server-based versus local use of Jenkins. We've ironed out some issues with our CI service so perhaps you can try again.

**Comment by melvyniandrag** [comment]
Thanks for the reply! It's probably the case that I just overlooked some part of the instructions. In any event, what I was referring to in my message was that when I made a tensorflow PR I got this kind of response from some bot automatically:
![automatedjenkins](https://user-images.githubusercontent.com/17210565/31301911-9ffaf9a2-aacb-11e7-812a-e8545e78f0f3.png)

There definitely are some issues though, because I passed the tests once, then an admin approved my merge, and then when the test suite ran again the failures I got were from some test hanging, not from broken code. I've never seen anything like this before and I imagine it might be very difficult to set up.
Best Regards,
Melvyn
